aws/auto-deployment.yaml

```diff
@@ -0,0 +1,232 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Auto deployment of Logz.io AWS Cost and Usage.
+Parameters:
+  S3BucketName:
+    Type: String
+    Description: >-
+      The name for the bucket which will contain the report files. The bucket name
+      must contain only lowercase letters, numbers, periods (.), and dashes (-),
+      and must follow Amazon S3 bucket restrictions and limitations.
+    Default: my-s3-bucket
+    AllowedPattern: '^[a-z0-9\-\.]*$'
+    MinLength: 1
+  ReportName:
+    Type: String
+    Description: >-
+      The name of report that you want to create. The name must be unique, is
+      case sensitive and can't include spaces.
+    Default: my-report
```

Comment:
Please change the default to AWS-Billing-Logz-Report

Reply:
Done